docs/ref/models/querysets.txt

```diff
@@ -1844,34 +1844,42 @@ they query the database each time they're called.
 .. method:: get(**kwargs)
 
 Returns the object matching the given lookup parameters, which should be in
-the format described in `Field lookups`_.
+the format described in `Field lookups`_. Normally you should use lookups
```

Comment:
I chopped `Normally`.

Reply:
I think there are use cases where the database doesn't have any uniqueness constraints but application level logic is relied on for unique results with given lookups. This even aplpies when using `order_with_respect_to`.

Reply:
Maybe we will chop also _"... by the database"_. This should cover all cases.

Reply:
That sounds reasonable and shorter. I've pushed the change.